content/en/user/profile.md

```diff
@@ -79,8 +79,11 @@ If you put an HTTPS link in your profile metadata, Mastodon checks if that link
 
 More precisely, Mastodon will validate the link under the following conditions:
 - Since 4.0: the hostname does not change after IDN normalization
+- the link to your Mastodon profile is added to a static HTML page used in your site. Websites served via Javascript-heavy content management systems such as StoryBlok often do not include static HTML, but check with your site developer
```

Comment:
This is inaccurate. All that is required is the rel=me attribute. Some CMSs may support this attribute, some may not. I haven't heard of a "StoryBlok" system before but it doesn't seem well known or common enough to mention explicitly, and I wouldn't want to give free advertising to some to some random proprietary product.

Reply:
See commit message. I was advised by my site developer to add the link to static HTML, and that's why validation is now working for me. Might as well guide users of CMS to a solution that could work! 

But I agree that there's no need to mention StoryBlok specifically.